src/main/java/io/mosip/digitalcard/util/CredentialUtil.java

```diff
@@ -0,0 +1,68 @@
+package io.mosip.digitalcard.util;
+
+import io.mosip.digitalcard.constant.ApiName;
+import io.mosip.digitalcard.dto.CredentialRequestDto;
+import io.mosip.digitalcard.dto.CredentialResponse;
+import io.mosip.digitalcard.dto.CredentialResponseDto;
+import io.mosip.digitalcard.dto.CredentialStatusResponse;
+import io.mosip.digitalcard.exception.ApiNotAccessibleException;
+import io.mosip.digitalcard.exception.ApisResourceAccessException;
+import io.mosip.digitalcard.exception.DigitalCardServiceException;
+import io.mosip.kernel.core.http.RequestWrapper;
+import io.mosip.kernel.core.http.ResponseWrapper;
+import io.mosip.kernel.core.util.DateUtils;
+import io.mosip.kernel.core.util.JsonUtils;
+import org.apache.commons.collections.map.HashedMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.net.URI;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class CredentialUtil {
+
+    @Autowired
+    private RestClient restClient;
+
+    @Autowired
+    Utility utility;
+
+    public CredentialResponse reqCredential(CredentialRequestDto dto) throws DigitalCardServiceException {
+        CredentialResponseDto credentialResponseDto=new CredentialResponseDto();
+        CredentialResponse credentialResponse=new CredentialResponse();
+        RequestWrapper<CredentialRequestDto> requestDto = new RequestWrapper<>();
+        try {
+            requestDto.setRequest(dto);
+            requestDto.setRequesttime(LocalDateTime.now());
+            ResponseWrapper<CredentialResponse> responseDto=restClient.postApi(ApiName.CREDENTIAL_REQ_URL,null,"","", MediaType.APPLICATION_JSON,requestDto,ResponseWrapper.class);
+            System.out.println(">>>>>>>>"+responseDto.toString());
```

Comment:
Please remove all the "System.out.println(" from code. use only log statements

Reply:
taken care